src/component/environments/EnvironmentTable/EnvironmentActionCell/EnvironmentActionCell.tsx

```diff
@@ -108,31 +102,22 @@ export const EnvironmentActionCell = ({
     const editId = useId();
     const deleteId = useId();
 
-    // Allow drag and drop if the user is permitted to reorder environments.
-    // Disable drag and drop while searching since some rows may be hidden.
-    const enableDragAndDrop = updatePermission && !searchQuery;
-
     return (
         <ActionCell>
-            <ConditionallyRender
-                condition={enableDragAndDrop}
-                show={
-                    <IconButton size="large">
-                        <DragIndicator titleAccess="Drag" cursor="grab" />
-                    </IconButton>
-                }
-            />
             <ConditionallyRender
                 condition={updatePermission}
                 show={
-                    <Tooltip title={toggleIconTooltip} arrow>
-                        <IconButton
-                            onClick={() => setToggleModal(true)}
-                            size="large"
-                        >
-                            <PowerSettingsNew />
-                        </IconButton>
-                    </Tooltip>
+                    <>
+                        <Tooltip title={toggleIconTooltip} arrow>
+                            <PermissionSwitch
```

Comment:
Same as above, the input is missing a label. There are some preexisting issues with the button labels on this page as well. I can take a look at those in a separate PR.